transport/cql_protocol_extension.cc

```diff
@@ -17,7 +17,8 @@ namespace cql_transport {
 
 static const std::map<cql_protocol_extension, seastar::sstring> EXTENSION_NAMES = {
     {cql_protocol_extension::LWT_ADD_METADATA_MARK, "SCYLLA_LWT_ADD_METADATA_MARK"},
-    {cql_protocol_extension::RATE_LIMIT_ERROR, "SCYLLA_RATE_LIMIT_ERROR"}
+    {cql_protocol_extension::RATE_LIMIT_ERROR, "SCYLLA_RATE_LIMIT_ERROR"},
+    {cql_protocol_extension::TABLETS, "SCYLLA_TABLETS"}
```

Comment:
Let's call it the same name as the the key in the custom map thing, so it's easier to grep across the codebase (also makes room for v2 if we add one).